### ISSUE TITLE
all_proficiencies=hero.base_proficiencies should only return non-hidden profs

### ISSUE THREAD

**Issue by elthran** [elthran/RPG-Game#299]
When you send all_profs to the profspage it should exclude hidden profs. Those are not modified or seen by the player and can only be given through items/abilities. An example might be "Fire cloak" which gives your hero 5 immolation damage each round of combat as a magic item. The proficiency immolation is given the value 5, but the player can't see this or edit it. It's just a value of the cloak.

**Comment by elthran** [comment]
Is this already possible by simply assigning a prof a None value instead of an attribute value? I assigned Thorns prof an attribute None and it seemed to create it and doesnt display it. That might be a simple solution? Since all hidden attributes would not have a linked attribute anyway (by design).

**Comment by klondikemarlen** [comment]
Currently hero has all profs even hidden ones. I suggest simply not giving them the hidden profs.
In hero.py -- __init__
```
for cls_name in proficiencies.ALL_CLASS_NAMES:
    # attributes.Attribute
    ProfClass = getattr(proficiencies, cls_name)
    prof = ProfClass()
    if not prof.hiden
        prof.hero = self
```

**Comment by klondikemarlen** [comment]
Fixed in a34d3c3aa6144
